### PR TITLE
GitHubSync update - release-8.4

### PR DESCRIPTION
**Pull request by particularbot6**
This is an automated synchronization PR.

The following source template repositories were used:
* particular/RepoStandards

### CODE REVIEW

File: src/NServiceBus.NHibernate/FodyWeavers.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Weavers>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Weavers xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="FodyWeavers.xsd">
```
Comment (bording) [inline src/NServiceBus.NHibernate/FodyWeavers.xml:2]:
```suggestion
<Weavers GenerateXsd="false">
```

File: src/NServiceBus.NHibernate/FodyWeavers.xsd [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (bording) [inline src/NServiceBus.NHibernate/FodyWeavers.xsd:1]: Delete this file.

**Review by bording — CHANGES_REQUESTED** [review]

File: src/NServiceBus.NHibernate/FodyWeavers.xsd (hunk 2 above)
Comment (SeanFeldman) [reply src/NServiceBus.NHibernate/FodyWeavers.xsd:1]: I constantly fall for this one. Thank you.

Comment (SeanFeldman) [reply src/NServiceBus.NHibernate/FodyWeavers.xsd]: Done

**Review by bording — APPROVED** [review]

**Pull request merged**